cypress-tests/cypress/e2e/adyen-card-flow-e2e-failure.cy.ts

```diff
@@ -0,0 +1,80 @@
+import * as testIds from "../../../src/Utilities/TestUtils.bs";
+import {
+  getClientURL,
+  createPaymentBody,
+  changeObjectKeyValue,
+} from "../support/utils";
+
+describe("Adyen Card payment flow test - Failure Cases", () => {
+  const publishableKey = Cypress.env("HYPERSWITCH_PUBLISHABLE_KEY");
+  const secretKey = Cypress.env("HYPERSWITCH_SECRET_KEY");
+  let getIframeBody: () => Cypress.Chainable<JQuery<HTMLBodyElement>>;
+  const iframeSelector =
+    "#orca-payment-element-iframeRef-orca-elements-payment-element-payment-element";
+  const adyenIframeSelector = ".adyen-checkout__iframe--threeDSIframe";
+
+  changeObjectKeyValue(createPaymentBody, "profile_id", Cypress.env("PROFILE_ID"));
```

Comment:
```suggestion
  changeObjectKeyValue(createPaymentBody, "profile_id", connectorProfileIdMapping.get(connectorEnum.ADYEN));
```

Reply:
Hi, I'm having problem running this command connectorProfileIdMapping it is not defined anywhere in commands or types.ts .. where do I need to import it from?

Reply:
Please take a pull from main